components/03-modules/gallery/gallery.hbs

```diff
@@ -0,0 +1,86 @@
+<div class="
+         gallery {{ class }}
+         {{#if typeHorizontal }}
+             gallery--horizontal
+         {{else}}
+             gallery--vertical
```

Comment:
can we delete one modifier here and pass it as default in function? Then we have only one modifier in code. Question related to others if/else modifiers like this.

Reply:
it's placeholder. As Iglo said it won't be probably used. And imho it make clear which version we are using and what styles.